examples/virtual-fields/src/seed/index.ts

```diff
@@ -0,0 +1,88 @@
+import payload from 'payload';
+import { MongoClient } from 'mongodb';
+import { eventsOne, eventsTwo } from './events';
+import { locationOne, locationTwo } from './locations';
+import { staffOne, staffTwo } from './staff';
+
+export async function seed() {
+  try {
+    payload.logger.info('Dropping database...');
```

Comment:
No need to drop the db here, just add `PAYLOAD_DROP_DATABASE=true` to your dev script.